Core/src/IfcGeometryConverter/FaceConverter.h

```diff
@@ -282,52 +282,113 @@ namespace OpenInfraPlatform {
 						\note	The \c IfcBSplineSurface is a subtype of \c IfcBoundedSurface.
 					*/
 
-				std::shared_ptr<carve::input::PolylineSetData> convertIfcBSplineSurface(const EXPRESSReference<typename IfcEntityTypesT::IfcBSplineSurface>& surface,
+				std::shared_ptr<carve::input::PolylineSetData> convertIfcBSplineSurface(
+					const EXPRESSReference<typename IfcEntityTypesT::IfcBSplineSurface>& surface,
 					const carve::math::Matrix& pos) const throw(...) 

```

Comment:
As discussed during our last meeting, make this a void and take in another parameter `itemData` - similarly to other convert functions.

Reply:
Done in commit 3ae78cd.